### ISSUE TITLE
deno only  shows autocomplete for https://x.nest.land

### ISSUE THREAD

**Issue by topperspal** [denoland/deno#18714]
A few days ago, everything was fine but When i created a new project and initialize deno workspace, it now only import autocomplete or import suggestions for  **https://x.nest.land**. It does not show auto suggestions for **https://deno.land** that is needed very all the time. Can anyone tell me about this if they have faced such an issue.

I also tried to set 
```json
"deno.suggest.imports.hosts": {
    "https://deno.land": true
}
```
but nothing happened. It still show suggestions for **https://x.nest.land**

**Comment by aapoalas** [comment]
Is this with VS Code?

**Comment by topperspal** [comment]
> Is this with VS Code?

Yes, I tried both VS Code and VS Code Insiders. Also uninstalled deno, restarted pc, installed deno again but still not working.

I am using a window p  if it matter.

**Comment by dsherret** [comment]
@topperspal when you go to `View > Output` then select "Deno Language Server" in the top right, does it show that import suggestions are enabled for https://deno.land? It should show something like:

> Enabling import suggestions for: https://deno.land

**Comment by topperspal** [comment]
I solved it by deleting the deno cache dir at **C:\Users\{{USERNAME}}\AppData\Local\deno**. I think I somehow, while navigating, edited a file in deno cache or may be something else has happened, but it works now pretty fine. ThankU for helping❤️🎉

**Comment by dsherret** [comment]
Glad you fixed it. I opened https://github.com/denoland/deno/issues/18745 to investigate the behaviour here. Probably it should at least purge the cache on deserialization error.

**Comment by elycheikhsmail** [comment]
I have similar issue

**Comment by topperspal** [comment]
> I have similar issue

try above [solution](https://github.com/denoland/deno/issues/18714#issuecomment-1512283237)

**Comment by elycheikhsmail** [comment]
> I solved it by deleting the deno cache dir at **C:\Users{{USERNAME}}\AppData\Local\deno**. I think I somehow, while navigating, edited a file in deno cache or may be something else has happened, but it works now pretty fine. ThankU for helping❤️🎉

i try this solution but don't work for me, I use npm: specifier in importmap like this
"express": "npm:express"

**Comment by dsherret** [comment]
@elycheikhsmail that’s a different issue.

Maybe https://github.com/denoland/deno/issues/15811 which hasn’t been implemented yet
